### PR TITLE
CMS ARS 5

### PR DESCRIPTION
**Pull request by karikarshivani**
Signed-off-by: Sumaa Sayed <ssayed1@umbc.edu>

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+title: Oracle Database 19c based on 19c CIS Benchmark
+maintainer: MITRE SAF Team
+copyright: MITRE, 2022
+copyright_email: saf@groups.mitre.org
+license: Apache-2.0
+summary: InSpec testing for overlay compliance with the Oracle 19c
+inspec_version: ">= 4.0"
+version: 1.0.0
+
+inputs:
```
Comment (karikarshivani) [inline inspec.yml:15]: Test to see if `sensitive` tag needs to be declared in the overlay

File: inspec.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+name: cms-ars-5.0-oracle-database-19c-cis-overlay
+title: Oracle Database 19c based on 19c CIS Benchmark
```
Comment (karikarshivani) [inline inspec.yml]: `name` and `title` can be the same value

File: inspec.yml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      - "oracle19c-2.2.15" # SC - 8
+      - "oracle19c-2.2.16" # SC - 8
+      - "oracle19c-5.2.1" # SC - 8
+      - "oracle19c-5.2.5" # SC - 8
+      - "oracle19c-5.2.6" # SC - 8
+      - "oracle19c-5.2.8" # SC - 8
+      - "oracle19c-4.5" # SC-28
+    <% end %>
+
+# oracle19c-2.2.10
```
Comment (karikarshivani) [inline inspec.yml]: No longer parameterized in the baseline so this can be removed

File: inspec.yml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    value: 5
+    <% end %>
+    profile: oracle-database-19c-cis-baseline
+
+# oracle19c-3.3
+  - name: password_life_time
+    description:
+    type: Numeric
+    value: 60
+    profile: oracle-database-19c-cis-baseline
```
Comment (karikarshivani) [inline inspec.yml]: An ERB condition is required for HVA before setting 60 as the value. No `elsif` required in this case as HML will get a caveat

File: inspec.yml (hunk 3 above)
Comment (ssayed118) [reply inspec.yml]: are you referring to the n/a array for low? or sec_max_failed_login_attempts?

File: inspec.yml (hunk 1 above)
Comment (karikarshivani) [reply inspec.yml:15]: Redeclared

File: inspec.yml [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+copyright: MITRE, 2022
+copyright_email: saf@groups.mitre.org
+license: Apache-2.0
+summary: InSpec testing for overlay compliance with the Oracle 19c
+inspec_version: ">= 4.0"
+version: 1.0.0
+
+depends:
+  - name: oracle-database-19c-cis-baseline
+    url: https://github.com/mitre/oracle-database-19c-cis-baseline/archive/parameterize.tar.gz
```
Comment (karikarshivani) [inline inspec.yml]: Branch name needs to be updated when the baseline PR is merged

Comment (karikarshivani) [reply inspec.yml]: Done

**Pull request merged**